### PR TITLE
Update hot reloading dependencies

### PR DESCRIPTION
**Pull request by gaearon**
## Motivation

These updates include important changes to correctness in the linked and underlying packages:

* https://github.com/gaearon/babel-plugin-react-transform/releases/tag/v2.0.1
* https://github.com/gaearon/babel-plugin-react-transform/releases/tag/v2.0.2
* https://github.com/gaearon/react-proxy/releases/tag/v1.1.3
* https://github.com/gaearon/react-proxy/releases/tag/v1.1.4
* etc

We need to include these if we want to ship hot reloading.

## Test Plan

Create a new project with those versions of packages.

screen shot 2016-03-06 at 18 49 20

Verify hot reloading still works.

## Reviewers

@martinbigio @skevy 

## Notes

I wasn’t sure whether to touch the shrinkwrap.

### CODE REVIEW

**Comment by martinbigio** [comment]
Awesome, thanks @gaearon!

**Comment by martinbigio** [comment]
@facebook-github-bot import

**Comment by skevy** [comment]
Thanks @gaearon :)

**Comment by gaearon** [comment]
Do I need to do anything here?

**Comment by skevy** [comment]
@gaearon nope. You shouldn't. Martin imported the change so that he can merge it manually. We do this with dependency changes so the shrinkwrap can be updated appropriately.

**Comment by satya164** [comment]
We should have the bot reply after import/shipit. cc @bestander @mkonicek

**Comment by gaearon** [comment]
:+1: Thanks.

**Comment by bestander** [comment]
Let me try again

**Comment by bestander** [comment]
@facebook-github-bot import

**Comment by bestander** [comment]
ok, so it was imported but Bot did not comment.
@martinbigio is comandeering it already

**Comment by martinbigio** [comment]
I've been trying to land this yesterday all day long but we've had some internal flakiness...
